### PR TITLE
allow spaces in polyfile

### PR DESCRIPTION
**Pull request by hixi**
Issues discovered by adding country to osmaxx.

Fixes:
- spaces in polyfile weren't handled properly

reviewed by:
- [ ] @das-g

### CODE REVIEW

File: tests/osm_test_helpers.py [hunk 1]
```diff
@@ -4,5 +4,5 @@
 from tests.conftest import test_data_dir
 
 # Monaco Polyfile
-POLYFILE_TEST_FILE_PATH = os.path.join(test_data_dir, 'osm', 'monaco.poly')
+POLYFILE_TEST_FILE_PATH = os.path.join(test_data_dir, 'osm', 'Monaco Containing Spaces.poly')
```
Comment (das-g) [inline tests/osm_test_helpers.py]: This constant isn't used anywhere.

File: tests/test_data/osm/monaco.poly [hunk 2]
```diff
@@ -1,31 +0,0 @@
...
-   7.435904E+00   4.374979E+01
-   7.437361E+00   4.375083E+01
-   7.436782E+00   4.375149E+01
-   7.437709E+00   4.375169E+01
-   7.438827E+00   4.375133E+01
-   7.438466E+00   4.374899E+01
-   7.441831E+00   4.374505E+01
-   7.448637E+00   4.373992E+01
-END
-END
```
Comment (das-g) [inline tests/test_data/osm/monaco.poly]: Did we have this file around twice?

Comment (hixi) [inline tests/test_data/osm/monaco.poly]: Yes.

**Comment by hixi** [comment]
@das-g can I merge this?

**Pull request merged**